webapp/stockstream/stockstream_api.py

```diff
@@ -86,14 +86,14 @@ def get_orders_by_id(order_ids):
         request = "/v1/orders?ids={}".format(",".join(chunk))
         votes = get_api_request(request)
         all_orders += votes
-    orders = sorted(all_orders, key=lambda k: k['timestamp'])
+    orders = sorted(all_orders, key=lambda k: k['created_at'])
```

Comment:
I realize these are tweaks to existing code but this kinda feels like basically the same code 4 times in a row.

Reply:
Yeah kinda ugly- though I can't think of a cleaner way to do this. The website needs orders to be sorted by their timestamp but the API service does not guarantee it- I don't think it makes sense to do it there either. I do play on refactoring this python stockstream client into an object-oriented pattern and can fix it up then.